docs/introduction/sla.md

```diff
@@ -0,0 +1,208 @@
+and the availability of the workloads.
+
+The Service Provider commits to inform the customer if this SLA is violated or
+a violation is anticipated. For this, email as a communication channel will be
+used. 
+
+A Customer contacts the CSC Service Desk for the case of a possible SLA
+violation. The case will be analysed internally and, if the violation is
+confirmed, CSC will inform the Customer about the reasons for the violation,
+planned mitigation actions and expected resolution time.
+
+## Limitations and constraints
+
+The provisioning of the service under the agreed service level targets is subject to the following limitations and constraints: 
+
+ * Support is provided primarily in the following languages: English, Finnish
+ * Downtimes caused due to upgrades for fixing critical security issues are not considered SLA violations. In the case of critical security upgrades CSC reserves the right to apply the upgrades with minimal notice
+ * A technical failure which affects individual customers does not count as downtime
+ * Failures of single instances of the workload (containers) are expected behaviour, and does not count as downtime
+ * Any other causes outside service provider’s direct control
+
+## Communication, reporting and escalation
+
+### General communication
+
+The following contacts will be generally used for communications related to the
+service in the scope of this SLA:
+
+| Communication channels ||
+--- | ---
+Contact for Customers | servicedesk@csc.fi
+Security incident reporting | (CSC’s Head of Security) security@csc.fi +358 (0) 94 57 2253
+
+### Reporting
+The service availability information is…. The information provided will be limited to service availability and by data security and privacy constraints.
+
+### Escalation and complaints
+
+For escalation and complaints, the defined contact point shall be used, and the
+following rules apply:
+
+1. First contact shall be established, preferably by e-Mail, to Contact for Customers address (See section 8.1) explaining the reason for the complaint with a sensible level of detail and clarity. Please also include, if possible, the following information:
+    * Name of the service
+    * Date and time of the events
+    * Usernames of affected users
+    * Channel to use on following communications (If other is preferred)
+2. CSC Service Desk will contact you back within three working days with information about the incident and which procedures will be adopted. 
+
+## Information security & data protection
+ CSC has approved a security policy and also follows security best practices.
+For CSC's customers, partners and staff there are detailed security guidelines.
+Many items in our security policies and guidelines refer to external compliance
+requirements. CSC also has procedures for risk and security management. For
+more information, please refer to the following pages:
+
+https://www.csc.fi/security
+
+https://www.csc.fi/web/research/rahti-security
```

Comment:
The Rahti security guidelines will be under the rahti.csc.fi domain and later under the docs.csc.fi domain.

Reply:
We'll have to go through all links to the Rahti docs. As afaik we have no Rahti security guidelines right now,  we can remove the link altogether.

Reply:
It's currently a work in progress by @juhanikataja . It might be that the security guidelines are finished in time for the SLA.